greedy_algorithms/jumpgame.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * @brief Implementation of an algorithm to solve the jumping game problem
```

Comment:
There is no reference to what exactly is this "jumping game problem"??
is this the link to the problem: https://leetcode.com/problems/jump-game/ ? If so, needs to be added in